pages/gall/[id]/index.tsx

```diff
@@ -24,6 +24,33 @@ type Props = {
     imagePaths: ImagePaths;
 };
 
+const sourceSortOptions = [
+    {
+        id: 0,
+        text: "Year ▲",
```

Comment:
Use single-quotes. This applies for all text values that are not part of an HTML control. 

Easiest to use the linter rules as defined in [.eslintrc.js](https://github.com/jeffdc/gallformers/blob/main/.eslintrc.js )and [.prettierrc.js](https://github.com/jeffdc/gallformers/blob/main/.prettierrc.js)

You can add a save hook to your editor to automatically apply the prettier rules on save so that you do not have to worry about them.

There are other lint violations that in the changes that will show up.